python/README.md

```diff
@@ -285,3 +264,9 @@ pip install -e .[test]
 # Run tests
 pytest -vvx
 ```
+
+If you need to debug or edit any .c files used to build the native extension, you can generate the `compile_commands.json` usable by most IDEs/clang tools using [Bear](https://github.com/rizsotto/Bear).
+
+```shell
+bear -- python setup.py build_ext --inplace && mv compile_commands.json build
+```
```

Comment:
BTW, this is a bit cryptic to me what this does / can help you with, but I am also not familiar with the tool or the compile_commands